dotnet/ios/ios13/SupportingMultipleWindowsOniPad/Gallery/View Controllers/GalleryViewController/GalleryViewController.cs

```diff
@@ -0,0 +1,166 @@
+﻿/*
+See LICENSE folder for this sample’s licensing information.
+
+Abstract:
+A view controller that displays a collection of photos.
+ It also shows how to create a new scene session via Drag and Drop.
+*/
+
+using ObjCRuntime;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Gallery;
+public partial class GalleryViewController : UIViewController, IUICollectionViewDataSource, IUICollectionViewDelegate, IUICollectionViewDragDelegate, IUICollectionViewDelegateFlowLayout {
+	#region Data
+
+	PhotoSection [] PhotoSections = Array.Empty<PhotoSection> ();
+
+	#endregion
+
+	#region Constructors
+
+	protected GalleryViewController (IntPtr handle) : base (handle)
+	{
+	}
+
+	#endregion
+
+	#region View controller lifecycle
+
+	public override void ViewDidLoad ()
+	{
+		base.ViewDidLoad ();
+		// Perform any additional setup after loading the view, typically from a nib.
+
+		PhotoSections = PhotoManager.SharedInstance.Sections;
+		Title = "Gallery";
+
+		galleryCollectionView.DataSource = this;
+		galleryCollectionView.Delegate = this;
+		galleryCollectionView.DragDelegate = this;
+		galleryCollectionView.RegisterNibForCell (UINib.FromName (GalleryCollectionViewCell.Key, null), GalleryCollectionViewCell.Key);
+	}
+
+	#endregion
+
+	#region Internal Functionality
+
+	bool TryGetPhoto (NSIndexPath indexPath, [MaybeNullWhen (false)] out Photo photo)
+	{
+		photo = null;
+
+		if (indexPath.Section >= 0 && indexPath.Section < PhotoSections.Length && indexPath.Row >= 0
+		&& indexPath.Row < PhotoSections[indexPath.Section].Photos.Length) {
+
+			if (PhotoSections[indexPath.Section].Photos[indexPath.Row] is null)
+				return false;
+
+			photo = PhotoSections[indexPath.Section].Photos[indexPath.Row];
+			return true;
```

Comment:
Can we simplify this? For example, if PhotoSections[indexPath.Section].Photos[indexPath.Row] is null it can be assigned to the out variable nevertheless, then we can return phot is not null
```suggestion
			photo = PhotoSections[indexPath.Section].Photos[indexPath.Row];
			return photo is not null;
```